### ISSUE TITLE
Tensorboard example outputs.

### ISSUE THREAD

**Issue by erogol** [mozilla/TTS#337]
Below is how a healty training looks like. I put it here as a reference for people using TTS.
![scalars_tb](https://user-images.githubusercontent.com/1402048/72343551-6fcc1f80-36cf-11ea-88a6-6c549ac824dc.PNG)
![healty_images_tb](https://user-images.githubusercontent.com/1402048/72343557-72c71000-36cf-11ea-8174-3b9fb49ae913.PNG)
![audios_tb](https://user-images.githubusercontent.com/1402048/72343561-7490d380-36cf-11ea-9b93-e770f8297bf7.PNG)

**Comment by IveJ** [comment]
How about your input data

**Comment by erogol** [comment]
@IveJ what do you like to know ?

**Comment by IveJ** [comment]
Hi Eren,

Could we minimize noise by tuning loss functions or another one, as
possible!?

Thanks.

**Comment by erogol** [comment]
All of these possible but I can't really comment without knowing all your data and training. Please post it on discourse.

**Comment by vcjob** [comment]
Hi @erogol 
according to the plots, what's the right step to stop training? Is it still learning at 300k+ step? what's the main plot we are looking at to decide when to stop? I mean, it seems like on the stop_loss plot it does not learn any longer, the training is going on though. Wouldn't it hurt the final model performance?

**Comment by erogol** [comment]
@vcjob https://github.com/mozilla/TTS/wiki/FAQ#how-do-i-know-when-to-stop-training

**Comment by WeberJulian** [comment]
Hi, I have a similar looking tensorboard but sometime on the ground truth, I have a weird artefact that is maybe just the silence trim but the models is struggling to replicate the green rectangle. Is this normal behaviour ? Shouldn't it be just deep blue if it's silence ?  

![Uploading Screenshot 2020-05-12 at 11.43.53.png…]()

**Comment by heoun** [comment]
@erogol - I'm running into an issue with tensorboard where "No scalar data was found." even though the output is the same as the one in the config.json. Any idea on how to resolve this issue? I have already ran the model and wanted to confirm the performance of my model via tensorboard.
